packages/storybook/stories/va-checkbox-uswds.stories.jsx

```diff
@@ -139,7 +141,9 @@ const IndeterminateTemplate = ({}) => {
   };
 
   const handleSelectAllToggle = event => {
-    const checkboxes = document.querySelectorAll('.example-checkbox');
+    const checkboxes = /** @type {HTMLInputElement[]} */ (
+      /** @type {unknown} */ (document.querySelectorAll('.example-checkbox'))
```

Comment:
Adding JSDOC type declaration to make "no `checked` prop" error go away. Due to a quirk in typing logic, has to first be cast as `unknown` before it can be cast as `HTMLInputElement[]`, but trust me I've done this before, tested it and it works. :)